frontendUiRootImpl/src/main/java/com/virtuslab/gitmachete/frontend/ui/impl/RediscoverSuggester.java

```diff
@@ -56,31 +51,25 @@ public void performIfNotDeclined() {
     }
   }
 
+  @UIEffect
   private void queueSuggestion(Path macheteFilePath) {
```

Comment:
I don't think anything is "queued" here anymore, it's a `@UIEffect` method, hence it's meant to execute synchronously

Reply:
Oh ok... see my comment below